R/fit_process.R

```diff
@@ -987,9 +987,20 @@ extract_severity <- function(samples) {
 
 extract_severity_region <- function(state, step, date) {
 
-  what <- c("ifr", "ihr", "hfr",
-            "ifr_strain_1", "ihr_strain_1", "hfr_strain_1",
-            "ifr_strain_2", "ihr_strain_2", "hfr_strain_2")
+  aggregated <- c("ifr", "ihr", "hfr")
+  disag <- c("ifr_disag_", "ihr_disag_", "hfr_disag_")
+  suffix <- paste0("_age_", c(sircovid:::sircovid_age_bins()$start))
```

Comment:
Avoid using unexported functions where possible. So export it, create a function within this package that does what you want, or you presumably just want `seq(0, 80, 5)` since we've never changed these age bands

Reply:
This is now addressed in commit https://github.com/mrc-ide/spimalot/pull/214/commits/d21ee110548605cffc950e3795f999c305a4eb60

Reply:
It's not been addressed here though